lib/rubocop/cop/correctors/alignment_corrector.rb

```diff
@@ -96,10 +96,6 @@ def each_line(expr)
           end
         end
 
-        def begins_its_line?(range)
-          (range.source_line =~ /\S/) == range.column
-        end
-
```

Comment:
Unrelated cleanup. It looks like during the Corrector class breakout, this method got duplicated. It already exists in `util.rb` which is included in this class.